PluginInstaller/Create-PluginInstaller.ps1

```diff
@@ -28,8 +28,8 @@ param(
   #$lib = "$wix\\WixUI.wixlib",
   $ui_ref = "WixUI_Mondo",
   $title = "UserPlugins",
-  $manufacturer = "Citrix",
-  $description = "XenCenter Plugins",
+  $manufacturer = $env:MANUFACTURER_NAME,
+  $description = $env:PRODUCT_NAME + " Plugins",
   $product_version = "1.0.0.0",
   $upgrade_code = "8282b90a-cb51-4c02-a1c1-ecfcff9861bf",
   $product_code = "8282b90a-cb51-4c02-a1c1-ecfcff9861bf",
```

Comment:
The value of $default_upgrade_code sort of like a place holder string and $upgrade_code as the default powershell parameter. It is inherited from the old XenCenter plugin template and definitely looks ugly. Do you want it to be fixed in this change or we separate a new one to do a thorough refactor(there are some other places also smell)?